tests/ui/places/test_places_gnd_tasks.py

```
@@ -0,0 +1,117 @@
+    assert "PID changed" in message
+    assert "040754766 -> 123456789" in message
+
+
+@mock.patch("rero_mef.places.gnd.tasks.requests_retry_session")
+def test_gnd_get_record_debug_reraises(mock_session, app):
+    """Test gnd_get_record reraises exception in debug mode."""
+    mock_session.return_value.get.side_effect = ValueError("Test error")
+
+    try:
+        gnd_get_record("040754766", debug=True)
+        assert False, "Should have raised ValueError"
```

Ruff (0.15.2)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)